x-pack/docs/en/security/configuring-stack-security.asciidoc

```diff
@@ -1,80 +1,152 @@
+-u elastic https://localhost:9200
+----
+// NOTCONSOLE
++
+`--cacert`::
+Path to the generated `http_ca.crt` certificate for the HTTP layer.
+
+*Next*: <<add-kib-user,Configure {kib} to connect to {es} with a password>>
 
 [discrete]
-[[security-basic-https-overview]]
-=== Basic security plus secured HTTPS traffic (Elastic Stack)
-
-This scenario builds on the one for basic security and secures all HTTP
-traffic with TLS. In addition to configuring TLS on the transport interface of
-your {es} cluster, you configure TLS on the HTTP interface for both
-{es} and {kib}.
-
-NOTE: If you need mutual (bidirectional) TLS on the HTTP layer, then you'll
-need to configure mutual authenticated encryption.
-
-You then configure {kib} and Beats to communicate with
-{es} using TLS so that all communications are encrypted. This level
-of security is strong, and ensures that any communications in and out of your
-cluster are secure.
-
-<<security-basic-setup-https,Set up basic security plus HTTPS traffic>>
-
-include::securing-communications/security-minimal-setup.asciidoc[]
-include::securing-communications/security-basic-setup.asciidoc[]
-include::securing-communications/security-basic-setup-https.asciidoc[]
-include::securing-communications/configuring-tls-docker.asciidoc[]
-include::securing-communications/enabling-cipher-suites.asciidoc[]
-include::securing-communications/tls-versions-jdk.asciidoc[]
-include::reference/files.asciidoc[]
-include::fips-140-compliance.asciidoc[]
+[[add-kib-user]]
+=== Configure {kib} to connect to {es} with a password
+
+With security enabled, users must log in to {kib} with a valid username and
+password.
+
+You'll configure {kib} to use the built-in `kibana_system` user and the password
+that you created earlier. {kib} performs some background tasks that require use
+of the `kibana_system` user.
+
+This account is not meant for individual users and does not have permission to
+log in to {kib} from a browser. Instead, you'll log in to {kib} as the `elastic`
+superuser. 
+
+. Add the `elasticsearch.username` setting to the `KIB_PATH_CONF/kibana.yml`
+file and set the value to the `kibana_system` user:
++
+[source,yaml]
+----
+elasticsearch.username: "kibana_system"
+----
++
+NOTE: The `KIB_PATH_CONF` variable is the path for the {kib}
+configuration files. If you installed {kib} using archive distributions
+(`zip` or `tar.gz`), the variable defaults to `KIB_HOME/config`. If you used
+package distributions (Debian or RPM), the variable defaults to `/etc/kibana`.
+
+. From the directory where you installed {kib}, run the following commands
+to create the {kib} keystore and add the secure settings:
+
+   a. Create the {kib} keystore:
++
+[source,shell]
+----
+./bin/kibana-keystore create
+----
+
+   b. Add the password for the `kibana_system` user to the {kib} keystore:
++
+[source,shell]
+----
+./bin/kibana-keystore add elasticsearch.password
+----
++
+When prompted, enter the password for the `kibana_system` user.
+
```

Comment:
Ping to add the relevant TLS config as discussed in slack

Reply:
Added the steps to configure TLS for Kibana in https://github.com/elastic/elasticsearch/pull/76052/commits/1a2da8d38bd7e5a28af78f409b25c732b43579ae